### ISSUE TITLE
Checkrain in constant loop or unable to jailbreak ios

### ISSUE THREAD

**Issue by peterpt** [checkra1n/BugTracker#2094]
Today i decided to rejail my iphone and i spend more than an hour trying to rejail it without any success .
Iphone was already jailbreak before so it supposed to be an easy step but it was connected for 2 months without a restart in non jailbreak mode .
I was only able to jailbreak it again at first time when i did a normal full reboot of the device .
I full rebooted the device , then i shut it down normally , and then i was able to start the process of jailbreak it again .

How to full reboot iphone instructions can be found here : 
https://support.apple.com/guide/iphone/force-restart-iphone-iph8903c3ee6/ios

My ios was 14.0

**Comment by UInt2048** [comment]
@peterpt You said in your profile that you'd close your account when Microsoft acquired GitHub - that was over 2 years ago...

Also, it seems like your problem's been fixed, so you might consider closing this issue.

**Comment by peterpt** [comment]
yup you are right , i did not closed up but i stopped to write tools in my account .
This is not an issue , this is an alert and probably a solution for many people that stay hours trying to rejail the ios without success .
It happened to me , so i decided to post it here for others .
